src/commands.rs

```diff
@@ -3537,14 +3537,24 @@ fn cmd_debug(
     Ok(())
 }
 
+// TODO: Move this somewhere where it can be reused by
+// `template_parser::SignatureTimestamp`.
 fn format_timestamp(timestamp: &Timestamp) -> String {
-    let utc = Utc
-        .timestamp(
-            timestamp.timestamp.0.div_euclid(1000),
-            (timestamp.timestamp.0.rem_euclid(1000)) as u32 * 1000000,
-        )
-        .with_timezone(&FixedOffset::east(timestamp.tz_offset * 60));
-    utc.format("%Y-%m-%d %H:%M:%S.%3f %:z").to_string()
+    let utc = match Utc.timestamp_opt(
+        timestamp.timestamp.0.div_euclid(1000),
+        (timestamp.timestamp.0.rem_euclid(1000)) as u32 * 1000000,
+    ) {
+        LocalResult::None => {
+            return "<out-of-range date>".to_string();
```

Comment:
Not `<out-of-range date>`?

Reply:
I'm confused. That's what it says, isn't it? Am I just missing some detail?

Reply:
How strange, Graphite rendered it as `"".to_string()`. Maybe some kind of HTML escaping bug on their end? 😬 (It also didn't recognize this file as Rust for some reason, but rather `undefined`.)


<img width="916" alt="Screen Shot 2022-11-14 at 13 17 16" src="https://user-images.githubusercontent.com/454057/201767331-7b0d565c-f0db-4fb4-a96d-2e59bd95d721.png">